lib/access_import/access_import.rb

```diff
@@ -20,8 +20,17 @@ def make_clients
       ).any?
       client_attrs = client_transformer.prepare_attributes(ac_client)
       client = Client.new(client_attrs)
+      client.created_at = ac_client[:d_Rollenbeginn]
+      client.updated_at = ac_client[:d_MutDatum]
       client.user = User.first
-      puts "Importing personen_rolle #{key} to Client.id: #{client.id}" if client.save!
+      next unless client.save!
```

Comment:
But how will I know for sure that save failed?
Because thats something I don't want to happen. It means I'm losing records I'm supposed to import.